xml/poc-install-deployer-node.xml

```diff
@@ -0,0 +1,215 @@
+   installation process is required for getting product updates and
+   for installing the SUSE OpenStack Cloud extension. Refer to the
+   <link xlink:href="https://www.suse.com/documentation/sles-12/book_sle_deployment/data/sec_i_yast2_conf_manual_cc.html"><citetitle>SUSE Customer Center Registration</citetitle></link>
+   section of the SUSE Linux Enterprise Server 12 SP4 Deployment Guide for further instructions.</para>
+  <para>Install a SLES12-SP4 server using the following values
+   (accepting all other defaults):</para>
+   <note>
+     <para>
+      Installing SUSE Linux Enterprise Server for SUSE OpenStack Cloud
+      requires the following steps, which are different from the default
+      SUSE Linux Enterprise Server installation process.
+     </para>
+   </note>
+    <screen>
+     IP Addressing:
+      Create a bond0 interface with eth0 and eth1 as the bond slaves
+      Hostname: deployer.example.com
+      bond0 IP: 192.168.200.10/24
+      Gateway: 192.168.200.1
+      DNS: 192.168.200.1 , 8.8.8.8
+     Partitioning:
+      /dev/sda1
+      Size: 250GB
+      Filesystem: ext4
+      Mount Point: /boot
+      /dev/sda2
+      Size: 250GB
+      Filesystem: vfat
+      Mount Point: /boot/efi
+      /dev/sda3
+      Size: (remaining space)
+      Type: LVM
+      LVM VG:
+      Name: ardana-vg
+      Disks: /dev/sda3
+      LVM LV:
+      Name: root
+      Size: 50GB
+      Filesystem: ext4
+     Users:
+      Username: root
+      Password: [PASSWORD]
+      Username: [USERNAME]
+      Password: [PASSWORD]
+    </screen>
+ </section>
+
+ <section xml:id="install-configure-SMT">
+   <title>Install and Configure the SMT Server</title>
+   <para>One way to provide the repositories needed to set up the nodes in
+     SUSE OpenStack Cloud is to install a Subscription Management Tool
+     (SMT) server on the Cloud Lifecycle Manager server, and then mirror
+     all repositories from SUSE Customer Center (SCC) via this server.</para>
+     <note>
+       <para>
+         This configuration is optional when deploying SUSE OpenStack Cloud.
+         However for the purpose of this proof of concept, this procedure is a
+         requirement.
+       </para>
+     </note>
+     <procedure>
+       <step>
+         <para>
+         Install the SMT Server pattern.
+         <screen>
+           $ sudo zypper in -t pattern smt
+         </screen>
+         </para>
+       </step>
+       <step>
+         <para>
+         Before you create a local mirror of the repositories, you need
+         appropriate organization credentials. You can get the
+         credentials from Novell Customer Center or SUSE Customer
+         Center (SCC), they are identical.
+         For more information on SMT configuration, see the
+         <link xlink:href="https://www.suse.com/documentation/suse-openstack-cloud-9/book_deployment/data/app_deploy_smt_config.html"><citetitle>SMT Configuration</citetitle></link>
```

Comment:
this also 404'd for me... maybe this will work when its live or something  that I'm overlooking?

Reply:
^^ same as above I think

Reply:
This will be the same (dash vs. underscore)